### PR TITLE
OSDOCS - CNV-38323: Reverting a bug fix description in the RNs

### PR DESCRIPTION
**Pull request by jherrman**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/CNV-38323

Link to docs preview:
https://78023--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 1]
```diff
@@ -286,6 +283,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //4.14 Leave in per Simone
 * Uninstalling {VirtProductName} does not remove the `feature.node.kubevirt.io` node labels created by {VirtProductName}. You must remove the labels manually. (link:https://issues.redhat.com/browse/CNV-22036[*CNV-22036*])
 
+//BZ 2151169
+* In a heterogeneous cluster with different compute nodes, virtual machines that have HyperV Reenlightenment enabled cannot be scheduled on nodes that do not support timestamp-counter scaling (TSC) or have the appropriate TSC frequency. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2151169[*BZ#2151169*])
```
Comment (dfitzmau) [inline virt/release_notes/virt-4-14-release-notes.adoc:287]:
```suggestion
* In a heterogeneous cluster with different compute nodes, virtual machines that have HyperV reenlightenment enabled cannot be scheduled on nodes that do not support timestamp-counter scaling (TSC) or have the appropriate TSC frequency. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2151169[*BZ#2151169*])
```
Lowercase according to [link](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/tlfs/datatypes/hv_partition_privilege_mask#:~:text=of%20guest%20debugging.-,AccessReenlightenmentControls,The%20partition%20has%20access%20to%20reenlightenment%20controls.,-CreatePartitions).

**Review by dfitzmau — APPROVED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

File: virt/release_notes/virt-4-14-release-notes.adoc (hunk 1 above)
Comment (jherrman) [reply virt/release_notes/virt-4-14-release-notes.adoc]: Good catch, fixed now

**Comment by jherrman** [comment]
> Hi @jherrman . The merger would need to know for sure if 4.15 is impacted. Can you confirm? Bug was fixed in 4.16?

@dfitzmau  SME confirmed in https://issues.redhat.com/browse/CNV-38323 that the issue impacts 4.15 and 4.16 as well (and possibly will for 4.17, too).

**Comment by dfitzmau** [comment]
Thanks, @jherrman . LGTM. WOuld you be OK to update the following description item to state "4.14+"?

Screenshot from 2024-07-22 16-15-19

**Comment by jherrman** [comment]
> Thanks, @jherrman . LGTM. WOuld you be OK to update the following description item to state "4.14+"?
> 

Done, thank you for the heads-up :-)

**Comment by jab-rh** [comment]
@jherrman, if this is intended for 4.14+, I think this should be a PR against the `main` branch and not the `enterprise-4.14` branch? Thanks!

**Comment by jherrman** [comment]
> @jherrman, if this is intended for 4.14+, I think this should be a PR against the `main` branch and not the `enterprise-4.14` branch? Thanks!

@jab-rh , my apologies, must have missed that in the OpenShift docs manual. Still, is it necessary? Changing the base branch to "main" complicates the PR quite a lot (due to the many added commits), and it's just a single very simple change, which should be easy to cherry-pick to other branches.

**Comment by kalexand-rh** [comment]
It's not one of the repo best practices to cherry-pick to main, but if you can find a merge reviewer who is willing to make sure that no other changes creep into other branches, you can.

**Comment by jherrman** [comment]
Thank you for the info Kathryn! 

However, one thing I did not realize earlier is that as an update to Release Notes, this cannot really be merged into Main. Also, as a bug fix description becoming a known issue description, it cannot be easily cherry-picked to the newer branches without conflict (since the newer branches don't contain the original bug fix).

So, this PR should still cover 4.14 (I changed the original description to reflect that) and I created a new one for 4.15 - 4.17: https://github.com/openshift/openshift-docs/pull/78023 - which should theoretically be cherry-pickable without much trouble

Hopefully this does the trick, apologies for the shenanigans...

**Comment by jab-rh** [comment]
@jherrman, can you squad the commits for this PR? Thank you!

**Review by sheriff-rh — APPROVED** [review]
LGTM, merging.

**Pull request merged**